collaboration/README.md

```diff
@@ -1,5 +1,86 @@
+- **Affective Trust**:  
+  We will cultivate trust by building personal connections through empathy, active listening, and support during challenges. Transparency, constructive feedback, and approachability will be key.  
+- **Addressing Damaged Trust**:  
+  Mistakes will be acknowledged with sincerity, responsibility will be taken, and open communication will be used to restore confidence.
+
+### 💬 Discussion & Debate  
+
+- **We will balance discussion and debate** by using each appropriately:  
+  - *Discussion*: For collaborative brainstorming, sharing diverse perspectives, and fostering inclusivity.  
+  - *Debate*: For resolving disagreements or evaluating competing ideas.  
+
+### 🙌 Respect  
+
+- We will ensure **mutual respect** by actively listening, giving constructive feedback, and recognizing contributions.  
+- Disrespect, such as interruptions or dismissive behavior, will be avoided to maintain a positive group dynamic.
+
+### 🌍 Inclusivity  
+
+- **We will foster inclusivity** by creating an environment where all voices are heard and valued. Respect for diverse perspectives and communication styles will ensure everyone feels included.
+
+### 🔄 Flexibility & Adaptability  
+
+- **We will remain flexible** by adjusting to individual circumstances and evolving group norms as needed. Feedback and project demands will guide these changes.
+
+### 📢 Communication  
+
+- **We will prioritize clear communication** through:  
+  - Slack for quick updates.  
+  - Email for formal documentation.  
+  - Virtual meetings for discussions and decisions.  
+- Agendas will keep meetings focused, and open discussions will encourage collaboration.
+
+### ⏰ Time Management  
+
+- **We will respect schedules and deadlines** by:  
+  - Allowing a 10-minute grace period for late arrivals.  
+  - Using pre-set agendas for productive meetings.  
+  - Coming prepared with completed tasks.
+
+### 🗳️ Decision-Making  
+
+- **We will strive for consensus** and use majority votes when disagreements arise.  
+- Major decisions will be documented and shared transparently.
+
+### 🛠️ Task Division & Collaboration  
+
+- **Tasks will be divided fairly** using a "divide and conquer" approach to ensure equal contributions.  
+- Progress will be tracked to maintain accountability, and contributions will be respected regardless of task complexity.
+
+### 🔗 Leadership  
+
+- **We will handle leadership flexibly** by assigning leaders as needed or sharing responsibilities among all members.
+
+### 💡 Conflict Resolution  
+
+- **We will address conflicts with openness and empathy**, focusing on collaborative resolutions. Mistakes will be acknowledged, and trust will be restored through accountability and dialogue.
+
+### 🚀 Norm Evolution  
+
+- **We will review and adapt norms** regularly as the team grows and challenges arise to keep them relevant and effective.
```

Comment:
I think there's a typo here, our team is already determined and will not be growing, other than that, everything seems perfect 👍🏻

Reply:
Actually, our team is still growing. If I remember right, Mr. Evan said, that if members in other groups ask to change groups with valid reasons, then they can be assigned to another group. So, it could be a possibility that a new member is added to our group. Hence validating the norm.

Reply:
Oh right, I forgot about that😅. Got it 👍🏻